GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -817,6 +817,10 @@ void Renderer::renderMaterialShader(RenderData* render_data,
                 shader_manager->getLightMapShader()->render(mvp_matrix,
                         render_data, curr_material);
                 break;
+			case Material::ShaderType::UNLIT_FBO_SHADER:
+                shader_manager->getUnlitFboShader()->render(
+                        mvp_matrix, render_data, curr_material);
```

Comment:
"mvp_matrix" should be "uniforms.u_mvp" post-multilight. Needs to be updated otherwise we will merge something that does not compile.